src/Core/TileMesh.js

```diff
@@ -101,6 +102,26 @@ class TileMesh extends THREE.Mesh {
             this.material.updateLayersUniforms();
         }
     }
+
+    /**
+     * Refresh a material of this tile, associated with a specific layer. Only
+     * works with ColorLayer for now.
+     *
+     * @param {ColorLayer} layer - The layer associated to the material.
+     * @param {View} [view] - The view to update, don't forget to call
+     * `view.notifyChange` otherwise.
+     */
+    refreshMaterial(layer, view) {
+        if (layer.isColorLayer) {
+            this.material.getLayer(layer.id).textures.forEach(t => Cache.delete(`${layer.source.uid}-${t.extent.toString('-')}`));
```

Comment:
The key cache could be factorized with the one done in the `dataSourceProvider`, because it's better for the maintainability of the code

Reply:
Yes, but if we merged #1264 there is no point to this, so let's wait and rebase on #1264 when it's done.